Sources/FluentUI_macOS/Components/DatePicker/CalendarHeaderView.swift

```diff
@@ -28,6 +28,66 @@ private struct Constants {
 	private init() {}
 }
 
+/// Allows the top row of the `CalendarHeaderView` to function as an accessible
+/// stepper for moving between months. Increment to move forwards, decrement to
+/// move backwards.
+class AccessibleCalendarHeaderStackView: NSStackView, NSAccessibilityStepper {
+	init(monthYearLabel: NSTextField, leadingButton: NSButton, trailingButton: NSButton) {
+		self.monthYearLabel = monthYearLabel
+		self.leadingButton = leadingButton
+		self.trailingButton = trailingButton
+
+		super.init(frame: .zero)
+
+		self.setAccessibilityElement(true)
+		self.setAccessibilityRole(.incrementor)
+
+		self.addView(monthYearLabel, in: .center)
+		self.addView(leadingButton, in: .leading)
+		self.addView(trailingButton, in: .trailing)
+	}
+
+	required init?(coder: NSCoder) {
+		fatalError("init(coder:) has not been implemented")
+	}
+
+	// MARK: NSAccessibilityStepper methods
+
+	override func accessibilityPerformIncrement() -> Bool {
+		return performStep(trailingButton)
+	}
+
+	override func accessibilityPerformDecrement() -> Bool {
+		return performStep(leadingButton)
+	}
+
+	override func accessibilityLabel() -> String? {
+		return monthYearLabel.accessibilityLabel()
+	}
+
+	override func accessibilityValue() -> Any? {
+		return monthYearLabel.accessibilityValue()
+	}
+
+	// MARK: Private
+
+	private func performStep(_ button: NSButton) -> Bool {
+		let isEnabled = button.isEnabled
+		if (isEnabled) {
+			button.performClick(button)
+			NSAccessibility.post(element: self, notification: .announcementRequested, userInfo: [
+				NSAccessibility.NotificationUserInfoKey.announcement: self.accessibilityValue() ?? "",
+				NSAccessibility.NotificationUserInfoKey.priority: NSAccessibilityPriorityLevel.medium.rawValue
+			])
+		}
```

Comment:
Is this still necessary or is there a way to get this for free since we're now a NSAccessibilityStepper?

Reply:
Unfortunately it does still seem necessary :(